src/MainTabScreen.tsx

```diff
@@ -10,7 +11,12 @@ const Tab = createBottomTabNavigator<Types.MainTabParams>();
 
 export const MainTabScreen = () => {
   return (
-    <Tab.Navigator initialRouteName={Types.Route.UserProfile}>
+    <Tab.Navigator
+      initialRouteName={Types.Route.UserProfile}
+      screenOptions={{
+        tabBarHideOnKeyboard: Platform.OS !== 'ios',
```

Comment:
not sure if it's needed

Reply:
> 
![Zrzut ekranu 2022-02-17 o 14 19 04](https://user-images.githubusercontent.com/13075882/154490029-f064e504-bb21-4f77-9a5f-6bd5bf9fb10d.png)


Do you have any suggestions how to fix it without hiding tabBar?
 I tried changing windowSoftInputMode but it didn't helped.

Reply:
maybe let's keep it true for both platforms then? :)

Reply:
with true on IOS i have problem with IOS keyboard ;(